### ISSUE TITLE
[COGNITION]: GIBCT Section 116 - VoiceOver is spelling out some all caps text instead of reading it

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#3538]
## Description

VoiceOver is spelling out some of the words in the Primary and Secondary SCO contacts list. This is caused by that text being in all caps instead of sentence case. I've set this as a very low priority item because I'm not sure of the data source or if it can be transformed.

Screenshot attached below.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

* As a screen reader user, I want to hear the contact names and titles read aloud, not spelled aloud.

## WCAG or Vendor Guidance (optional)

* [Language of Parts: Understanding SC 3.1.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/meaning-other-lang-id.html)

## Screenshots or Trace Logs

![Screen Shot 2019-11-18 at 12.09.32 PM.png](https://images.zenhubusercontent.com/5ac217b74b5806bc2bcd3fc8/af754a6b-d886-41ed-86f8-d954b693c0c7)

**Comment by WillMcCormack** [comment]
This section is on staging however it will not be deployed to Production for this launch.

**Comment by shawkey-daniel** [comment]
The source data for school certifying officials is providing the data in all caps. I am reluctant to try to modify case programmatically. I think it should be controlled at the source.

**Comment by 1Copenut** [comment]
@shawkey-daniel Agreed. I figured the data was coming in as all caps. AFAIK, there's not a good way to downcase with CSS alone, so it's a data source question.

**Comment by desireeturner** [comment]
@1Copenut Is this something you'd like to keep open? If so, should we discuss a path forward on resolving this?

**Comment by desireeturner** [comment]
@1Copenut we just released an update to this section and while I don't think our changes would have resolved this issue we did reduce the content displayed in the section. It doesn't seem we've received any reports from users about it since it was opened in November 2019, so we'd like to go ahead and close it.

**Comment by 1Copenut** [comment]
@desireeturner Given the level of effort and the upstream dependency, I'm okay closing this issue. Ideally data sources don't store values in all caps, but this one's outside any of our control. I'll go ahead and close with a label "Won't fix". Thank you!
